modules/openy_features/openy_prgf/modules/openy_prgf_small_banner/openy_prgf_small_banner.features.yml

```diff
@@ -0,0 +1,4 @@
+excluded:
+  - core.entity_view_display.taxonomy_term.color.color
+  - core.entity_view_mode.taxonomy_term.color
```

Comment:
We don't need excluded section here, because those view modes will live in own module and won't be listed as dependencies.